File: MOC Develop SQL 2005/Labfiles/Lab08/Solution/cs/WindowsDataClient/Form1.cs
#region Using directives

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Windows.Forms;

#endregion

namespace WindowsDataClient
{
	partial class Form1: Form
	{
		public Form1()
		{
			InitializeComponent();
		}

        private void Form1_Load(object sender, EventArgs e)
        {
            getProductsTableAdapter.Fill(productsDS);
        }

        private void dataNavigatorSaveItem_Click(object sender, EventArgs e)
        {
            getProductsTableAdapter.Update(productsDS);
        }    
	}
}

File: MOC Develop SQL 2005/Labfiles/Lab08/Solution/vb/WindowsDataClient/Form1.vb
Public Class Form1

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        GetProductsTableAdapter.Fill(ProductsDS)
    End Sub

    Private Sub dataNavigatorSaveItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles dataNavigatorSaveItem.Click
        GetProductsTableAdapter.Update(ProductsDS)
    End Sub
End Class
